### PR TITLE
fix(core): peek db row should always show the related db row property

### PR DESCRIPTION
**Pull request by pengx17**
fix BS-1963
depends on https://github.com/toeverything/blocksuite/pull/8818

### CODE REVIEW

**Comment by pengx17** [comment]
* **#8942** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8942?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8942?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `canary`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Comment by pengx17** [comment]
### Merge activity

* **Dec 4, 3:52 AM EST**: [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/8942) disabled "merge when ready" on this PR due to: a merge conflict with the target branch; resolve the conflict and try again..
* **Dec 6, 4:47 AM EST**: A user added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=AFFiNE).
* **Dec 6, 5:17 AM EST**: A user  merged this pull request with the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=AFFiNE).

File: packages/frontend/core/src/modules/editor/entities/editor.ts [hunk 1]
```diff
@@ -157,6 +157,7 @@ export class Editor extends Entity {
         if (params.databaseId && params.databaseRowId) {
           const defaultOpenProperty: DefaultOpenProperty = {
             type: 'database',
+            docId: params.databaseId,
```
Comment (forehalo) [inline packages/frontend/core/src/modules/editor/entities/editor.ts]: wrong？

**Review by L-Sun — APPROVED** [review]

File: packages/frontend/core/src/modules/editor/entities/editor.ts (hunk 1 above)
Comment (pengx17) [reply packages/frontend/core/src/modules/editor/entities/editor.ts]: unsubscribing cursor

Comment (forehalo) [reply packages/frontend/core/src/modules/editor/entities/editor.ts]: anyway，cursor is not me

Comment (pengx17) [reply packages/frontend/core/src/modules/editor/entities/editor.ts]: turns out this section is incorrect. the params for dbid/rowid is for scrolling to the current doc's db row, not to opening the doc's db properties

**Pull request merged**